wrangler-core/src/main/java/io/cdap/directives/column/ChangeColCaseNames.java

```diff
@@ -94,5 +104,40 @@ public Mutation lineage() {
       .all(Many.of())
       .build();
   }
+  @Override
+  public Relation transform(RelationalTranformContext relationalTranformContext,
+                            Relation relation) {
+    java.util.Optional<ExpressionFactory<String>> expressionFactory = getExpressionFactory(relationalTranformContext);
```

Comment:
nit: add import for Optional separately, change in other places too as necessary